Doc/glossary.rst

```diff
@@ -988,6 +995,16 @@ Glossary
       interface can be registered explicitly using
       :func:`~abc.register`.
 
+   set
+      An unordered collection of distinct :term:`hashable` objects that eases
```

Comment:
Question: Although it would be handy for newcomers to know some usages of a `set`, should this be considered part of the definition? 

I am refering to this part:

 >eases duplicate removal and computation of mathematical operations such as intersection and difference.

Reply:
That's a pretty good question, actually. It's hard to understand what actually is expected from a set type (not the specific set class) from the docs, as it's mostly mixed with set (class) and frozenset:

https://docs.python.org/3/library/stdtypes.html#set-types-set-frozenset

Perhaps this is something I should ask on python-dev ?

Reply:
Thanks folks. Perhaps reorganizing a bit would help:
```
An unordered collection of distinct :term:`hashable` objects that were introduced in :pep:`218`.

Python provides two built-in set types: set (a :term:`mutable` type), and frozen set (an :term:`immutable` type). See :ref:`types-set`.
```

Reply:
Because of Raymon's comment on the BPO issue, I modified the PR substantially, to the point that it only adds a link, actually :sweat_smile: